docs/UserGuide.md

```diff
@@ -184,6 +184,101 @@ Examples:
 * `filter b/70-79 ap/01/01/2025 - 01/01/2026`
 * `filter b/70-79 t/medication t/Dr Tan`
 
+### Importing contacts: `filter`
+
```

Comment:
Should the command be `import` here?

Reply:
Thanks for spotting it! Have fixed it in the latest commit.